websites/A/Apple/presence.ts

```diff
@@ -461,9 +464,229 @@ presence.on("UpdateData", async () => {
+    } else if(urlpath[1] === 'ios' || urlpath[1] === 'ipados' || urlpath[1] === 'tvos' || urlpath[1] === 'watchos' || urlpath[1] === 'macos' || urlpath[1] === 'mac-catalyst' || urlpath[1] === 'xcode' || urlpath[1] === 'swift' || urlpath[1] === 'testflight' || urlpath[1] === 'swift-playgrounds' || urlpath[1] === 'app-clips') {
+      presenceData.details = document.querySelector("h2.localnav-title>a")?.textContent || "Apple Developer";
+
+      if(!urlpath[2])
+        presenceData.state = 'Overview';
+      else if(urlpath[2] === 'whats-new')
+        presenceData.state = 'What\'s new';
+      else if(urlpath[2] === 'submit')
+        presenceData.state = 'Submissions';
+      else if(urlpath[1] === 'macos' && urlpath[2] === 'iphone-and-ipad-apps')
+        presenceData.state = 'iPhone and iPad apps';
+      else if(urlpath[1] === 'macos' && urlpath[2] === 'distribution')
+        presenceData.state = 'Distributing software';
+      else if(urlpath[1] === 'watchos' && urlpath[2] === 'features')
+        presenceData.state = 'Features';
+      else
+        presenceData.state = document.querySelector("h1.typography-headline")?.textContent || "Other";
+    } else if(urlpath[1] === 'documentation') {
+      const page = document.querySelector("span.current.item");
+
+      presenceData.details = 'Documentation';
+      presenceData.state = page?.textContent || "Home";
+
+      if (setting.buttons && page) {
+        presenceData.buttons = [
+          {
+            label: `View Page`,
+            url: window.location.href
+          }
+        ];
+      }
+    } else if(urlpath[1] === 'design') {
+      presenceData.details = "Design";
+
+      if(!urlpath[2])
+        presenceData.state = 'Overview';
+      else if(urlpath[2] === 'whats-new')
+        presenceData.state = 'What\'s new';
+      else if(urlpath[2] === 'human-interface-guidelines')
+        presenceData.state = 'Human Interface Guidelines';
+      else if(urlpath[2] === 'resources')
+        presenceData.state = 'Resources';
+      else if(urlpath[1] === 'testflight')
+        presenceData.state = "Testflight";
+      else
+        presenceData.state = document.querySelector("h1.typography-headline")?.textContent || "Other";
+    } else if(urlpath[1] === 'safari' || urlpath[1] === 'app-store-connect' || urlpath[1] === 'business' || urlpath[1] === 'app-store' || urlpath[1] === 'education' || urlpath[1] === 'classkit' || urlpath[1] === 'programs') {
+      presenceData.details = document.querySelector("h2.localnav-title>a")?.textContent || "Apple Developer";
+      presenceData.state = document.querySelector("a.localnav-menu-link.link-"+cpage)?.textContent || document.querySelector("span.localnav-menu-link.current")?.textContent || "Other";
+    } else if(urlpath[1] === 'games')
+      presenceData.state = "Games";
+    else if(urlpath[1] === 'forums') {
+      const buttons = {
+        show: false,
+        label: "View Page",
+        url: window.location.href
+      };
```

Comment:
I'm literally confused why you have something like `show`. If you don't want to show this, why you just don't delete this const??

Reply:
![image](https://user-images.githubusercontent.com/53608074/118506457-9ec42c80-b72d-11eb-9e9d-33eb4830f00b.png)
So that I don't have this 3 times

Reply:
I'm confused af. You only set `show` to false on this const